lib/search-bar/index.js

```diff
@@ -118,14 +118,16 @@ export default class SearchBarContainer extends Component {
         multiSection: multiSection
       });
       clearTimeout(keyPressTimeout);
-      keyPressTimeout = setTimeout(() => {
-        const analyticsObject = {
-          event: 'searchTyping',
-          searchInput: value,
-          suggestions: this.props.autocompleteOptions ? this.props.autocompleteOptions.length : 0
-        };
-        dataLayer.push(analyticsObject);
-      }, 3000);
+      if (value && this.state.value === value && value !== '') {
```

Comment:
`value !== ''` is implicit from the first condition (truthy `value`) so is redundant.

Also, what is the intention of the second clause?